src/datreant/core/collections.py

```diff
@@ -620,6 +620,59 @@ def __getitem__(self, index):
             out = super(Bundle, self).__getitem__(index)
         return out
 
+    def get(self, *tags, **categories):
+        """Retrieve all Treants which match the defined tags and categories
+
+        :Arguments:
+             *tags*
+                tuple of tags to match
+             *categories*
+                  key: value pairs
+
+        :Returns:
+             *bundle*
+                 A bundle containing all matches from within this bundle
+
+
+        :Examples:
+
+          bundle.get(length=5.0, colour='green')
+
+        is equivalent to::
+
+          bundle.categories.groupby(['length', 'colour'])[5.0, 'green']
+
+
+          bundle.get('this')
+
+        is equivalent to
+
+          bundle.tags.filter('this')
+
+
+          bundle.get('this', length=5)
+
+        is equivalent to::
+
+          bundle.tags.filter('this').categories.groupby('length')[5.0]
+        """
+        if not (tags or categories):
```

Comment:
this is so that `bundle.get()` returns nothing, rather than everything.  I think this is the most logical return for this?  I was thinking if someone was making the arguments to `.get` programmatically, then if they searched for nothing.. they should get nothing?